proto/feature/service.proto

```diff
@@ -38,9 +38,10 @@ import "proto/feature/strategy.proto";
 import "proto/feature/target.proto";
 
 message GetFeatureRequest {
-  string id = 1;
+  string id = 1 [(google.api.field_behavior) = REQUIRED];
   reserved 2;
-  string environment_id = 3;
+  string environment_id = 3 [(google.api.field_behavior) = REQUIRED];
+  google.protobuf.Int32Value feature_version = 4;
```

Comment:
We could have a description here.

E.g., When the version is not set, it will return the latest flag.